### PR TITLE
Replace Peter Eberlein with Florian Mueller

### PR DESCRIPTION
**Pull request by petereberlein**

### CODE REVIEW

**Comment by duglin** [comment]
@chipchilders can you add `fmui` as a collaborator on this repo? I think that'll make PullApprove happy.

**Comment by chipchilders** [comment]
invite sent @duglin @fmui

**Comment by duglin** [comment]
@chipchilders thanks

**Comment by avade** [comment]
LGTM

**Comment by petereberlein** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Comment by avade** [comment]
@petereberlein - tl;dr please send an email to the committers. We will need 4/7 of PMC members to approve Florian (excluding yourself).

> New Dedicated Committers can be added to the PMC, based on a majority vote of the current Dedicated Committers within the PMC. The PMC should take into account the contributions of that individual to the PMCs projects.
>
> Votes regarding changes to Dedicated Committers must be done electronically via a private email thread including all Dedicated Committers, to try and avoid any negative consequences to the person in question through public archival of the discussion.

https://www.openservicebrokerapi.org/governance

**Comment by angarg12** [comment]
LGTM
